src/com/company/ppj11/Tasks45.java

```diff
@@ -0,0 +1,41 @@
+package com.company.ppj11;
+
+import java.util.Arrays;
+
+public class Tasks45 {
+    public static void main(String[] args) {
+        task4();
+        task5();
+    }
+
+    private static void task4() {
+        int[] array = {1, 2, 3, 4, 5};
+        int tmp;
+        for (int i = 0; i < array.length - 1; i++) {
+            tmp = array[i];
```

Comment:
why this `tmp` value is assigned inside the loop?

Reply:
fixed